submissions/NadiaVorontsova/hooli-style-popup/index.html

```diff
@@ -0,0 +1,238 @@
+                                                <p>Setting</p>
+                                            </div>
+                                        </a>
+                                    </li>
+                                    <li class="header__popup__menu__item">
+                                        <a href="#" class="header__popup__menu__link link">
+                                            <div class="header__popup__menu__container">
+                                                <img src="assets/mic.png" width="48" height="35" alt=""
+                                                    class="header__popup__menu__img">
+                                                <p>Microphone</p>
+                                            </div>
+                                        </a>
+                                    </li>
+                                    <li class="header__popup__menu__item">
+                                        <a href="#" class="header__popup__menu__link link">
+                                            <div class="header__popup__menu__container">
+                                                <img src="assets/sale.webp" width="48" height="35" alt=""
+                                                    class="header__popup__menu__img">
+                                                <p>Sale</p>
+                                            </div>
+                                        </a>
+                                    </li>
+                                    <li class="header__popup__menu__item">
+                                        <a href="#" class="header__popup__menu__link link">
+                                            <div class="header__popup__menu__container">
+                                                <img src="assets/collections.webp" width="48" height="35" alt=""
+                                                    class="header__popup__menu__img">
+                                                <p>Collections</p>
+                                            </div>
+                                        </a>
+                                    </li>
+                                    <li class="header__popup__menu__item">
+                                        <a href="#" class="header__popup__menu__link link">
+                                            <div class="header__popup__menu__container">
+                                                <img src="assets/books.webp" width="48" height="35" alt=""
+                                                    class="header__popup__menu__img">
+                                                <p>Books</p>
+                                            </div>
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
```

Comment:
What this for so much free space?)
![image](https://user-images.githubusercontent.com/16196199/183475568-b62cdd7a-64ea-4ad3-925a-f661cc21459d.png)

Reply:
I fixed it